### PR TITLE
SM-31 loader

### PR DESCRIPTION
**Pull request by RobertS-ki**

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -1,14 +1,35 @@
...
+          },
+        },
+      },
+    },
     screens: {
       sm: '576px',
       md: '768px',
       lg: '1024px',
       xl: '1280px',
       '2xl': '1536px',
```
Comment (n0macx) [inline tailwind.config.js:31]: 2xl without quotes perhaps ?

File: src/features/homePage/HomePage.tsx [hunk 2]
```diff
@@ -1,5 +1,13 @@
+import { Loader } from '../Loader';
+
+Loader;
```
Comment (vieraboschkova) [inline src/features/homePage/HomePage.tsx]: Is this necessary?

**Comment by vieraboschkova** [comment]
That is one amazing loader!!!!

File: src/features/homePage/HomePage.tsx (hunk 2 above)
Comment (RobertS-ki) [reply src/features/homePage/HomePage.tsx]: It's interesting how it got pass eslint/prettier/husky :)

**Review by KatarzynaChmielecka — DISMISSED** [review]

**Review by KatarzynaChmielecka — APPROVED** [review]

**Pull request merged**